proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -27,6 +27,24 @@ In order for the app to be always registered, the changes in SDL session establi
 
 ![registeringApp_flow](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow.png)
 
+1. The developer will set their `requiresAudioSupport` to true.
+2. The library will send a `StartService` for the RPC service with a new param in the payload, `requiresAudioSupport`.
+3. Core will receive the `StartService` for the RPC service:
+3-1. If `requiresAudioSupport` was set to false or not set, it will continue in flow.
+3-2. Core will check it's current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
+3-3. If there is no Audio support, core will check if it supports the auto connect BT function. If it does it will continue in flow.
+3-4. If `requiresAudioSupport` is true and there is no supported audio methods and Core does not support the auto connect BT function, it will send a `StartServiceNAK` with the reason `No audio support available`.
```

Comment:
Please fix the formatting of these sub-bullets.  See attached for how they're currently displayed.  These should each have their own line and be indented under Section 3.
![Screen Shot 2020-04-23 at 10 59 06 AM](https://user-images.githubusercontent.com/22600496/80114523-b09ac680-8551-11ea-8fc3-901b5f8a1deb.png)

Reply:
This is not yet resolved.  Please see attached screenshots.  You may need to add line breaks to make sure the sub-bullets are on their own line and indented.
![Screen Shot 2020-04-27 at 9 12 50 AM](https://user-images.githubusercontent.com/22600496/80376276-94f03280-8867-11ea-9f9f-cb9a50f62107.png)